### PR TITLE
Fix event image/venue map logic

### PR DESCRIPTION
**Pull request by csebianlander**
This PR removes several fields and moves another in order to simplify and clarify the logic around selecting a venue image and/or generated map for an Event page. It also fixes an issue where a large CFPB logo image was shown for all events once concluded, regardless of selections made in any of these fields.

---

## Removals

- Unused but visible `archive_image` field
- Unnecessary `post_event_image` field and related unit test

## Changes

- Moved `venue_image` tab to the General Content tab and relabeled

## How to test this PR

1. localhost
2. Create or edit an Event page
3. Use the moved "Image" fields in the General Content tab. Note that maps don't appear to render on localhost, but you will see a change in the spacing of the image area that indicates an attempt to generate

## Notes and todos

- This is not intended to be a perfect or complete solution to Event page woes but instead an iteration that fixes a longstanding problem and removes some cruft. There are many other changes we _could_ integrate into a PR like this, but I think we should wait for any larger-scale changes to code until we have a more focused effort to review it.

## Checklist

- [ ] PR has an informative and human-readable title
  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Future todos are captured in comments and/or tickets
- [ ] Project documentation has been updated, potentially one or more of:
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### CODE REVIEW

File: cfgov/v1/models/learn_page.py [hunk 1]
```diff
@@ -316,20 +322,6 @@ class EventPage(AbstractFilterPage):
...
-                FieldPanel("venue_image"),
-            ],
-            heading="Venue Image",
-        ),
-        MultiFieldPanel(
-            [
-                FieldPanel("post_event_image_type"),
-                FieldPanel("post_event_image"),
-            ],
-            heading="Post-event Image",
```
Comment (willbarton) [inline cfgov/v1/models/learn_page.py:331]: We should remove these fields from the model and not just from the admin panels, if they're not used, on lines 247-267 above here. A quick query suggests there's one `EventPage` with `post_event_image_type` of `"image"`.

File: cfgov/v1/jinja2/v1/events/_macros.html [hunk 2]
```diff
@@ -128,27 +125,13 @@ <h2 class="event-venue__heading">
...
-                             src="{{ static('img/cfpb_video_cover_card_1380x776.png') }}"
-                             alt="">
-                    {% endif %}
+                {% elif event_state == 'past' and event.archive_video_id %}
+                    {# DISPLAY VIDEO RECORDING #}
+                    {% import 'v1/includes/organisms/video-player.html' as video_player with context %}
+                    {{ video_player.render(
+                        video_id=event.archive_video_id,
+                        button_pos='bottom-right'
+                    ) }}
```
Comment (willbarton) [inline cfgov/v1/jinja2/v1/events/_macros.html:134]: With this change, the placeholder image will never display after an event is over. I assume that's the large CFPB logo you referenced? 

Right now it looks like we have 96 events where the `post_event_image_type` is `"placeholder"` and `archive_video_id` is null that will show this, i.e. https://www.consumerfinance.gov/about-us/events/archive-past-events/field-hearing-on-medical-debt-collection/

image

**Review by willbarton — CHANGES_REQUESTED** [review]
I am not entirely familiar with the event logic, but I think the main thing is, if we're not using the `post_event_image` fields anymore, I think we should remove them.

File: cfgov/v1/jinja2/v1/events/_macros.html (hunk 2 above)
Comment (csebianlander) [reply cfgov/v1/jinja2/v1/events/_macros.html:134]: Yeah, the idea is to remove the "placeholder" altogether and only show it if it's explicitly added (now via the `venue_image` field).

**Pull request merged**